core/mount/mount_linux.go

```diff
@@ -92,6 +92,15 @@ func (m *Mount) mount(target string) (err error) {
 		options   = m.Options
 	)
 
+	// X-containerd.* options are internal mount options that should be processed
+	// by mount transformers (e.g., mkdir, mkfs) before reaching this layer.
+	// If any remain here, it indicates the transformer failed to consume them.
+	for _, opt := range options {
+		if strings.HasPrefix(opt, "X-containerd.") {
+			return fmt.Errorf("internal mount option %q was not consumed by transformer", opt)
```

Comment:
`internal mount option %q was not consumed by the mount manager`?
since a mount handler like the erofs mount handler later can handler `X-containerd.` too